game-core/src/test/resources/Total_World_War_Dec1941.xml

```diff
@@ -4,6 +4,66 @@
   <info name="Total World War: December 1941" version="2.7.7.2"/>
   <triplea minimumVersion="1.8"/>
   <diceSides value="12"/>
+  <variableList>
+    <variable name="AntiTankTargets">
+      <element name="germanMech.Infantry"/>
+      <element name="germanTank"/>
+      <element name="germanHeavyTank"/>
+      <element name="russianMech.Infantry"/>
+      <element name="russianTank"/>
+      <element name="russianHeavyTank"/>
+      <element name="americanMech.Infantry"/>
+      <element name="americanTank"/>
```

Comment:
Side-note, I suspect part of the problem here is that we do not model a 'unit' type or a unit class. For example, if we said 'anti-tank' targets is any unit of class "tank", then we could likely simplify:

```
<unitType name="Tank" unitClass="tank">
    <players>
        <player>German</player>
         :
         :
    </players>
</unit>

<unitType name="HeavyTank" unitClass="tank">
    <players>
        <player>German</player>
         :
         :
    </players>
</unit>
```

Reply:
Correct. The issue is one of the problems this feature is addressing but is doing it in a more generic fashion so it can address other challenges as well.